modules/fedimint-ln-common/src/db.rs

```diff
@@ -125,11 +127,18 @@ impl_db_lookup!(
 );
 
 #[derive(Debug, Encodable, Decodable, Serialize)]
-pub struct BlockHeightKey;
+pub struct BlockHeightVoteKey(pub PeerId);
```

Comment:
Do we want this to be backwards compatible? This will require a database migration

Reply:
Probably not until we do a release.